### PR TITLE
chore(operator): sync cstor and NDM operator with latest changes

### PR DESCRIPTION
**Pull request by prateekpandey14**
- Sync service account name changes in cstor-operator
- Sync NDM operator yaml with latest changes

Signed-off-by: prateekpandey14 <prateek.pandey@mayadata.io>

### CODE REVIEW

File: deploy/ndm-operator.yaml [hunk 1]
```diff
@@ -36,58 +507,93 @@ data:
...
       hostNetwork: true
-      # host PID is used to check status of iSCSI Service when the NDM
-      # API service is enabled
-      #hostPID: true
+      hostPID: true
+      serviceAccountName: openebs-maya-operator
       containers:
       - name: node-disk-manager
-        image: openebs/node-disk-manager:1.2.0
+        image: openebs/node-disk-manager:ci
```
Comment (prateekpandey14) [inline deploy/ndm-operator.yaml:589]: should we use `1.4.1` latest release or `ci` is fine ?

Comment (akhilerm) [reply deploy/ndm-operator.yaml:589]: `ci` is fine as we  maintain the released operators in the charts repo

File: deploy/ndm-operator.yaml [hunk 2]
```diff
@@ -36,58 +507,93 @@ data:
...
+      # nodeSelector:
+      #   "openebs.io/nodegroup": "storage-node"
+      # Use host network as container network to monitor udev source using netlink
+      # to detect disk attach and detach events using fd.
       hostNetwork: true
-      # host PID is used to check status of iSCSI Service when the NDM
-      # API service is enabled
-      #hostPID: true
+      hostPID: true
+      serviceAccountName: openebs-maya-operator
```
Comment (akhilerm) [inline deploy/ndm-operator.yaml:586]: The service account needs to be changed here.

File: deploy/ndm-operator.yaml [hunk 3]
```diff
@@ -164,77 +667,104 @@ spec:
...
     matchLabels:
-      name: openebs-ndm-operator
-      openebs.io/component-name: ndm-operator
+      name: node-disk-operator
+  template:
+    metadata:
+      labels:
+        name: node-disk-operator
+    spec:
+      serviceAccountName: openebs-maya-operator
```
Comment (akhilerm) [inline deploy/ndm-operator.yaml:688]: service account needs to be changed

File: deploy/ndm-operator.yaml [hunk 4]
```diff
@@ -164,77 +667,104 @@ spec:
...
+---
+# Create NDM cluster exporter deployment.
+# This is an optional component and is not required for the basic
+# functioning of NDM
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ndm-cluster-exporter
+  namespace: openebs
+spec:
```
Comment (akhilerm) [inline deploy/ndm-operator.yaml:737]: Is the exporter required?

**Review by akhilerm — COMMENTED** [review]
given some comments,

File: deploy/ndm-operator.yaml (hunk 4 above)
Comment (prateekpandey14) [reply deploy/ndm-operator.yaml:737]: no we can remove that, i did remove the exporter daemonset but missed this.

File: deploy/ndm-operator.yaml (hunk 2 above)
Comment (prateekpandey14) [reply deploy/ndm-operator.yaml]: done

**Review by akhilerm — APPROVED** [review]
LGTM

**Review by shubham14bajpai — APPROVED** [review]
LGTM

**Pull request merged**